api.py

```diff
@@ -22,9 +22,7 @@ async def status():
 
 logger = get_logger()
 
-AJV_HOST = os.getenv("AJV_HOST", "localhost")
-
-AJV_VALIDATOR_URL = f"http://{AJV_HOST}:5002/validate"
+AJV_VALIDATOR_URL = os.getenv("AJV_VALIDATOR_URL", "http://localhost:5002/validate")
```

Comment:
I know we didn't have this documented before but this seems like our oversight, since we only have this single env var in Validator. Since we want to "update any documentation/Readme with the above" as part of this task maybe it is worth documenting this env var in the `## Running` part of the README?

Reply:
Yeah i was wondering where to add it. I have added a section under `Running`. Let me know if it needs any changes 👍